frontend/test/metabase/scenarios/dashboard/dashboard-drill.cy.spec.js

```diff
@@ -506,6 +506,41 @@ describe("scenarios > dashboard > dashboard drill", () => {
       });
     });
   });
+
+  it("should not hide custom formatting when click behavior is enabled (metabase#14597)", () => {
```

Comment:
Also, when you write a repro (even as a part of the fix), please tag your PR with `support/Repro` and (even more important) pretty please give `.Reproduced` label to the fixed issue. 
Thank you 🙏